packages/react-ui-core/src/Gmap/FreeDrawLayer.js

```diff
@@ -0,0 +1,92 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import autobind from 'autobind-decorator'
+import themed from 'react-themed'
+
+@themed(['FreeDraw', 'FreeDraw_ToolTip'], { pure: true })
+export default class FreeDrawLayer extends PureComponent {
+  static propTypes ={
+    map: PropTypes.object,
```

Comment:
Should we check if the `map` prop is null before retrieving its dom in `map.getDiv()` below in multiple places? Or perhaps give the map prop a default value if needed?